docs/team/app-guests.md

```diff
@@ -0,0 +1,43 @@
+title: Sematext App Guests
+description: App Sharing is restricted only to a particular App. Nothing besides the shared App is accessible to the invited user for example, dashboards are at the account level can can thus be shared only through Account Sharing.
+
+App guests is **restricted only to a particular App**. Nothing besides the 
+shared App is accessible to the invited user. 
+
+*__Note__: Dashboards are at the account level and can thus be shared only through Account Members.*
+
+This option is useful if you want to be restrictive about which Apps can be 
+seen by others or what kind of effect they have on your team. 
+
+By only configuring App guests, guests **can't see or edit alert rules** created 
+by your team, they **can't use your team's notification hooks**, meaning they're 
+**limited only to the App they have access to**.
+
+To share one of your Apps with other users, go to the 
+[App guests screen](https://apps.sematext.com/ui/team/app-guests) in Sematext Cloud.
+
+<video style="display:block; width:100%; height:auto;" controls autoplay loop>
+  <source src="https://cdn.sematext.com/videos/app-guests.mp4" type="video/mp4" />
+</video>
+
+## Apps you can access
+
+To see the list of all the Apps you can access click on the **Apps** section. 
+
+![Apps You Can Access](../images/team/apps_you_can_access.png)
+
+If you have a pending invite a red circle is displayed next to the **Team** menu item in the main navigation 
```

Comment:
If you have a pending invite a red circle is displayed next to the **Team** menu item in the main navigation 

=>

If you have a pending invite, a red circle is displayed next to the **Team** menu item in the main navigation